integration-tests-support/kafka/src/main/java/org/apache/camel/quarkus/test/support/kafka/KafkaTestResource.java

```diff
@@ -18,96 +18,38 @@
 
 import java.util.Collections;
 import java.util.Map;
-import java.util.function.Function;
 
-import com.github.dockerjava.api.exception.NotFoundException;
 import io.quarkus.test.common.QuarkusTestResourceLifecycleManager;
 import io.strimzi.test.container.StrimziKafkaContainer;
-import org.apache.camel.quarkus.test.FipsModeUtil;
 import org.eclipse.microprofile.config.ConfigProvider;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
-import org.testcontainers.containers.ContainerFetchException;
-import org.testcontainers.containers.GenericContainer;
-import org.testcontainers.images.builder.ImageFromDockerfile;
 import org.testcontainers.utility.TestcontainersConfiguration;
 
 public class KafkaTestResource implements QuarkusTestResourceLifecycleManager {
     protected static final String KAFKA_IMAGE_NAME = ConfigProvider.getConfig().getValue("kafka.container.image", String.class);
     private static final Logger LOGGER = LoggerFactory.getLogger(KafkaTestResource.class);
 
     private StrimziKafkaContainer container;
-    private GenericContainer j17container;
 
     @Override
     public Map<String, String> start() {
         LOGGER.info(TestcontainersConfiguration.getInstance().toString());
 
         try {
-            startContainer(KAFKA_IMAGE_NAME, name -> new StrimziKafkaContainer(name));
+            container = new StrimziKafkaContainer(KAFKA_IMAGE_NAME)
+                    /* Added container startup logging because of https://github.com/apache/camel-quarkus/issues/2461 */
+                    .withLogConsumer(frame -> System.out.print(frame.getUtf8String()))
```

Comment:
Nitpick - can we use a proper logger for log consumer instead of writing directly to STDOUT.

Reply:
definitiley, I just reverted the change, without noticing this one

Reply:
fixed, Thanks for the review!